### PR TITLE
my commit Anton

### LINKED ISSUE #14: Refactor student call
- ToDo description

### PR DESCRIPTION
**Pull request by MaksAnton**

### CODE REVIEW

**Comment by KseniiaPelykh** [comment]
Hey! thank you very much for your PR :) Good job! 
Could you please remove changes for package.json and package-lock.json?

File: routes/students.js [hunk 1]
```diff
@@ -34,4 +34,8 @@ router.get('/kott', (req, res, next) => {
     res.render('students/kott', null)
 })
 
+router.get('/kott', (req, res, next) => {
```
Comment (fspv) [inline routes/students.js]: The route should be unique for every user and match what's written in `res.render`

**Comment by MaksAnton** [comment]
error correction

**Comment by fspv** [comment]
LGTM

@KseniiaPelykh is there anything else to be done here?

**Comment by KseniiaPelykh** [comment]
> LGTM
> 
> @KseniiaPelykh is there anything else to be done here?

Nope, it's perfect! Thanks!

**Pull request merged**